scripts/analysis/amwg_table.py

```diff
@@ -149,10 +151,13 @@ def amwg_table(adf):
         #Save the baseline to the first case's plots directory:
         output_locs.append(output_locs[0])
 
-    #-----------------------------------------
+    #Declare any derived quantities here:
+    derived_vars = {"RESTOM":["FSNT", "FLNT"]}
```

Comment:
That would probably be best, in case the derived quantities are desired in other plots/scripts. After looking into it quickly it seems like this might be a larger update, so since the time series plot doesn't really depend on it, I'll open an issue and fix that in a separate PR.